src/AutoSizedImage.js

```diff
@@ -1,8 +1,5 @@
 import React, {PureComponent} from 'react';
```

Comment:
All sources moved to a separate folder to separate them from repo spam and simplify the babel command